records.go

```diff
@@ -13,6 +13,7 @@ import (
 type Records struct {
 	origins []string // for easy matching, these strings are the index in the map m.
 	m       map[string][]dns.RR
+	fallthru bool
```

Comment:
Please use "github.com/coredns/coredns/plugin/pkg/fall" to implement fallthrough function.

Reply:
Cool, I didn't know this existed, I'll take a look. Thanks!